io-engine/src/bin/io-engine.rs

```diff
@@ -89,6 +89,11 @@ fn start_tokio_runtime(args: &MayastorCliArgs) {
     if !ENABLE_NEXUS_RESET.load(Ordering::SeqCst) {
         warn!("Nexus reset is disabled");
     }
+    if args.lvm {
+        env::set_var("LVM", "1");
+        env::set_var("LVM_SUPPRESS_FD_WARNINGS", "1");
```

Comment:
Probably, we shouldn't override this var, if it was set explicitly to "0".
Also, should we document somewhere that we override the environment?

Reply:
Done